packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js

```diff
@@ -9,15 +9,27 @@ export default /* @ngInject */ ($stateProvider) => {
     redirectTo: (transition) =>
       transition
         .injector()
-        .getAsync('backup')
-        .then((backup) => {
-          if (backup.isInactive()) {
-            return { state: 'app.dedicatedClouds.datacenter.backup.new' };
+        .getAsync('licence')
+        .then(({ isSplaActive }) => {
+          if (!isSplaActive) {
+            return {
+              state: 'app.dedicatedClouds.datacenter.backup.spla-licence',
+            };
           }
-          if (backup.isLegacy()) {
-            return { state: 'app.dedicatedClouds.datacenter.backup.legacy' };
-          }
-          return false;
+          return transition
```

Comment:
Use Promise.all to make both async calls together. Right now, they are called one after another. These two calls can be executed parallelly to save time

refer to https://github.com/ovh/manager/packages/manager/modules/pci/src/projects/project/streams/onboarding/onboarding.routing.js